concrete-core/src/specification/engines/cleartext_conversion.rs

```diff
@@ -0,0 +1,35 @@
+use super::engine_error;
+use crate::specification::engines::AbstractEngine;
+use crate::specification::entities::CleartextEntity;
+
+engine_error! {
+    CleartextConversionError for CleartextConversionEngine @
+}
+
+/// A trait for engines converting cleartexts.
+///
+/// # Semantics
+///
+/// This [pure](super#operation-semantics) operation generates a cleartext containing the
+/// conversion of the `input` cleartext to a different representation.
+///
+/// # Formal Definition
```

Comment:
Yes, it is a placeholder to put more formal definitions (which we are expected to do promptly...) Maybe it is better to hide those though, because it is unclear whether we will be able to do it before next version.